scrapy/utils/reactor.py

```diff
@@ -57,6 +58,10 @@ def install_reactor(reactor_path, event_loop_path=None):
     reactor_class = load_object(reactor_path)
     if reactor_class is asyncioreactor.AsyncioSelectorReactor:
         with suppress(error.ReactorAlreadyInstalledError):
+            if sys.version_info >= (3, 8) and sys.platform == "win32":
+                policy = asyncio.get_event_loop_policy()
```

Comment:
Should we also load coverage data from Windows runs?

Reply:
Ideally, but I don’t think you need to worry about that as part of this pull request unless you want to. Maybe open a separate issue to address that eventually?